.archon/commands/defaults/archon-implement.md

```
@@ -93,19 +93,38 @@ Provide a valid plan path or GitHub issue containing the plan.
 
 ### 2.2 Branch Decision
 
-| Current State     | Action                                               |
-| ----------------- | ---------------------------------------------------- |
-| In worktree       | Use it (log: "Using worktree")                       |
-| On base branch, clean    | Create branch: `git checkout -b feature/{plan-slug}` |
-| On base branch, dirty    | STOP: "Stash or commit changes first"                |
-| On feature branch | Use it (log: "Using existing branch")                |
+**IMPORTANT: Never switch branches or create new branches. The isolation system has already set up the correct branch for this worktree. Work on whatever branch you are currently on.**
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)